acceptance/Gemfile

```diff
@@ -1,21 +1,26 @@
 source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
 def location_for(place, fake_version = nil)
-  if place =~ /^(git:[^#]*)#(.*)/
-    [fake_version, { :git => $1, :branch => $2, :require => false }].compact
-  elsif place =~ /^file:\/\/(.*)/
-    ['>= 0', { :path => File.expand_path($1), :require => false }]
+  git_match = place.match(/^(git:[^#]*)#(.*)/)
+  file_match = place.match(%r{^file://(.*)})
+  if git_match
+    git, branch = git_match.captures
```

Comment:
this is off by one, the first element of captures is the whole string, so you want git to be git_match[1] and branch to be git_match[2]

Reply:
oh this is wrong captures takes care of that